### ISSUE TITLE
need ability to Toggle LHA info between decimal and minutes for fractions of a degree

### ISSUE THREAD

**Issue by johnbellora** [Stellarium/stellarium#1344]
need ability to Toggle LHA info between decimal and minutes for fractions of a degree.

John Bellora

**Comment by alex-w** [comment]
LHA only or all HA?

**Comment by johnbellora** [comment]
Hello,
all would be nice but I use GHA and LHA hundreds of times a night. HA would be good because I will be using that just as much in the near future. I will be making another donation shortly because this is valuable to me and I greatly respect your work and appreciate all the authors to have an open dialog to add features that are helpful to the  users.
Best RegardsJohn Bellora


-----Original Message-----

**Comment by johnbellora** [comment]
all
Thank you


-----Original Message-----

**Comment by axd1967** [comment]
@johnbellora out of curiosity - how do you use these GHA/LHA values? e.g. do you enter them in a computerised mount? or do you process these values into other data?

**Comment by johnbellora** [comment]
Hello,
Thank you for asking. It is celestial Navigation program I am writing, The program will allow the user to have no previous experience in Celestial Navigation to quickly determine longitude with one sky measurement star to star. I already works but I am using your software for verification as well as actual star measurements from my back yard. I pick a location in your program and choose all different dates and times and compare the LHA to my program  and converting the response the minutes to decimal in every measurement is a pain in the butt. It is called Stellar Navigation because it does not use the horizon but rather only stars relative position. Thus the name Stellar. Most boaters have a phobia of celestial navigation and usually quiet shortly into the course. I came up with an easier method. 
I wished I could program like all of you but I cannot. I would love a collaboration if possible because I would want somewhat limited graphic. Mine will be very crude. I have my own navigational stars, 18 in the northern Hemisphere and about 18 in the southern.


-----Original Message-----

**Comment by axd1967** [comment]
Maybe the "use decimal degrees" option might suffice to be extended.

![image](https://user-images.githubusercontent.com/3529789/98030926-0827c200-1e12-11eb-8b91-9ef38d569f34.png)

**Comment by alex-w** [comment]
> Maybe the "use decimal degrees" option might suffice to be extended.

This option is not used within plugin, because navigational data show as in the navigational books.

@johnbellora should the plugin following option "Use decimal degrees" from the main GUI or it should be as separate (specific for plugin) option? Should this option enable using decimal degrees for all navigational data?

**Comment by johnbellora** [comment]
Hello,
Yes when switching from the main GUI  would like to extent that to include to the LHA & GHA calculations. I am hoping for that greatly. the others would be great but maily LHA & GHA.
Thank you for taking time to listen and consider
John.


-----Original Message-----

**Comment by johnbellora** [comment]
Hello,
Yes you can see the LHA, Decimal is enabled but the LHA & GHA is in degrees, & Minutes, The Minutes have a decimal but I would like the minutes converted the tenths of a degree instead of minutes which decimal for fractions of a minute. Example. from below screen shotLHA= 359°45.7' converting to decimal is 359.761666°
This makes fas for calculations which I have hundreds a night to convert
Best RegardsJohn Bellora

-----Original Message-----

**Comment by alex-w** [comment]
Please check version 0.20.3-50e9bca0fd: https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Comment by johnbellora** [comment]
I found it and I was able to use the Random location. The appears to be great. Once I take my readings and I believe I have the correct Longitude and latitude, How do I uncover the real location to verify my results to see how close I came?
Just a suggestion once in the navigation mode. The mouse is set to azimuth and altitude readings on the screen which I set it to. This is great because it is used as the sextant. 
It would be fun if the mouse pointer was an image of a little sextant. I used to program long ago and in basic language I was able to draw anything I wanted to be used for the mouse pointer. Also I think Windows 3.11 used to allow you to choose a BMP file and that was used as the mouse pointer. Just a thought! Should i suggest this on GITHUB?

Best regards
John Bellora


-----Original Message-----
